pygmt/x2sys.py

```diff
@@ -0,0 +1,289 @@
+
+    fmtfile : str
+        Format definition file prefix for this data set [See Format Definition
+        Files for more information]. Specify full path if the file is not in
+        the current directory.
+
+        Some file formats already have definition files premade. These include:
+
+        - mgd77 (for plain ASCII MGD77 data files)
+        - mgd77+ (for enhanced MGD77+ netCDF files)
+        - gmt (for old mgg supplement binary files)
+        - xy (for plain ASCII x, y tables)
+        - xyz (same, with one z-column)
+        - geo (for plain ASCII longitude, latitude files)
+        - geoz (same, with one z-column).
+
+    suffix : str
+        Specifies the file extension (suffix) for these data files. If not
+        given we use the format definition file prefix as the suffix (see
+        *fmtfile*).
+
+    discontinuity : str
+        ``d|g``
+        Selects geographical coordinates. Append **d** for discontinuity at the
+        Dateline (makes longitude go from -180 to + 180) or **g** for
+        discontinuity at Greenwich (makes longitude go from 0 to 360
+        [Default]). If not given we assume the data are Cartesian.
+
+    spacing : str or list
+         ``dx[/dy]``
+         x_inc [and optionally y_inc] is the grid spacing. Append **m** to
+         indicate minutes or **s** to indicate seconds for geographic data.
+         These spacings refer to the binning used in the track bin-index data
+         base.
+
+    units : str or list
+        ``d|sunit``.
+        Sets the units used for distance and speed when requested by other
+        programs. Append **d** for distance or **s** for speed, then give the
+        desired unit as:
+
+        - **c** - Cartesian userdist or userdist/usertime
+        - **e** - meters or m/s
+        - **f** - feet or feet/s
+        - **k** - km or kms/hr
+        - **m** - miles or miles/hr
+        - **n** - nautical miles or knots
+        - **u** - survey feet or survey feet/s
+
+        Default is ``units=["dk", "se"]`` (km and m/s) if *discontinuity* is
+        set, and ``units=["dc", "sc"]`` otherwise (Cartesian units).
+
+    {R}
+    {V}
+
+    gap : str or list
+        ``t|dgap``.
+        Give **t** or **d** and append the corresponding maximum time gap (in
+        user units; this is typically seconds [Infinity]), or distance (for
+        units, see *units*) gap [Infinity]) allowed between the two data points
+        immediately on either side of a crossover. If these limits are exceeded
+        then a data gap is assumed and no COE will be determined.
+
+    {j}
+    """
+    with Session() as lib:
+        arg_str = " ".join([tag, build_arg_string(kwargs)])
+        lib.call_module(module="x2sys_init", args=arg_str)
+
+
+@fmt_docstring
+@use_alias(
+    A="combitable",
+    C="runtimes",
+    # D="",
+    I="interpolation",
+    R="region",
+    S="speed",
+    T="tag",
+    Q="coe",
+    V="verbose",
+    W="numpoints",
+    # Z="",
```

Comment:
Ok, almost done aliasing all the arguments for `x2sys_cross`. Any good names for [`D`](https://docs.generic-mapping-tools.org/6.1/supplements/x2sys/x2sys_cross.html#d) or [`Z`](https://docs.generic-mapping-tools.org/6.1/supplements/x2sys/x2sys_cross.html#z)?

Reply:
My bad suggestions:

- D (True/False, N or S) = overridepole
- Z (True/False) = rawreport/rawvalue/xvalue